src/router/routes.ts

```diff
@@ -1,5 +1,14 @@
-import { AmalgamationTypes, FilingTypes, RouteNames } from '@/enums'
+import { RouteNames } from '@/enums'
 import * as Views from '@/views'
+import {
+  AmalgamationRegularRoutes,
+  AmalgamationShortRoutes,
+  DissolutionFirmRoutes,
+  DissolutionRoutes,
+  IncorporationRoutes,
+  RegistrationRoutes,
+  RestorationRoutes
+} from './routes/index'
```

Comment:
This file was getting long so I broke it up.

Reply:
Some upcoming merge conflicts coming here depending on who merges first 😆

Reply:
Age before beauty. I insist 😆

Reply:
Hahahaha, OK with me 🤣